src/main/webapp/app/exercises/shared/feedback/item/programming-feedback-item.service.ts

```diff
@@ -96,32 +66,28 @@ export class ProgrammingExerciseFeedbackItemService implements FeedbackItemServi
      * @param showTestDetails
      * @private
      */
-    private createProgrammingExerciseScaFeedbackItem(feedback: Feedback, showTestDetails: boolean): FeedbackItem {
+    private createScaFeedbackItem(feedback: Feedback, showTestDetails: boolean): FeedbackItem {
         const scaCategory = feedback.text!.substring(STATIC_CODE_ANALYSIS_FEEDBACK_IDENTIFIER.length);
         const scaIssue = StaticCodeAnalysisIssue.fromFeedback(feedback);
         const text = showTestDetails ? `${scaIssue.rule}: ${scaIssue.message}` : scaIssue.message;
-        const previewText = computeFeedbackPreviewText(text);
 
         return {
-            type: FeedbackItemType.Issue,
-            category: this.translateService.instant('artemisApp.result.detail.codeIssue.name'),
+            type: 'Static Code Analysis',
```

Comment:
Why the change to a string instead of an enum?

Reply:
It's a string literal union type, functionally allowing us the same type safety as with an enum but with less typing and the aforementioned advantage in dev tools